content/news/2025-08-02-qml-project.md

```diff
@@ -0,0 +1,63 @@
+
+![Preview](https://raw.githubusercontent.com/mixxxdj/proposals/main/proposals/contents/2024-11-16_qml_interface_overview_live.svg)
+![Preview](https://raw.githubusercontent.com/mixxxdj/proposals/main/proposals/contents/2024-11-16_qml_interface_library_split_button.svg)
+![Preview](https://raw.githubusercontent.com/mixxxdj/proposals/main/proposals/contents/2024-11-16_qml_interface_waveform_overview_customization.png)
+
+## Why We Are Redesigning Mixxx
+
+The current user interface of Mixxx has served us well, but as technology evolves, so must we. The shift to QML will simplify customization and maintenance, allowing us to focus more on innovative features and less on upkeep. This transition will also enable us to phase out our homemade theme system, streamlining our development process.
+
+## Key Features of the New Design
+
+1. **Modernized Library and Waveform Views**: We are introducing a fresh look for the Library and Waveform views, making them more intuitive and user-friendly.
+
+![Preview](https://raw.githubusercontent.com/mixxxdj/proposals/main/proposals/contents/2024-11-16_qml_interface_overlays_playlist.svg)
+
+2. **Enhanced Support for Lightweight Platforms**: The new design will be optimized for tablets, smartphones, and touchscreen laptops, ensuring a seamless experience across all devices.
+
+3. **Improved Search Capabilities**: The new search experience will offer better usability of advanced search options, making it easier to find and manage your tracks.
+
+![Preview](https://raw.githubusercontent.com/mixxxdj/proposals/main/proposals/contents/2024-11-16_qml_interface_overlays_search.svg)
+![Preview](https://raw.githubusercontent.com/mixxxdj/proposals/main/proposals/contents/2024-11-16_qml_interface_overlays_search3.svg)
+
+4. **Interactive Settings and Preferences**: We are introducing more interactive settings for audio routing, controllers, and waveforms, providing a more dynamic and user-friendly experience.
+
+![Preview](https://raw.githubusercontent.com/mixxxdj/proposals/main/proposals/contents/2024-11-16_qml_interface_setting_audio_routing_advanced.svg)
+![Preview](https://raw.githubusercontent.com/mixxxdj/proposals/main/proposals/contents/2024-11-16_qml_interface_setting_controller_overview.svg)
+![Preview](https://raw.githubusercontent.com/mixxxdj/proposals/main/proposals/contents/2024-11-16_qml_interface_setting_controller_pointnclick.svg)
+![Preview](https://raw.githubusercontent.com/mixxxdj/proposals/main/proposals/contents/2024-11-16_qml_interface_setting_waveform.svg)
+
+
+## Get Involved
+
+We are calling on developers and testers to join us in this exciting journey. Whether you are a seasoned developer or a passionate DJ looking to contribute, there are many ways you can help:
+
+## For Developers
+
+- **Code Contributions**: Help us implement the new QML interface, fix bugs, and add new features. Your expertise in Qt/QML will be invaluable as we transition to this new technology.

```

Comment:
Let's also say that this is a big project that will require a lot of help, and this is a great opportunity to have a big impact on the ground floor of a big effort on a popular project in a way that will give developers valuable experience.  Let's sell people on the opportunities that working on mixxx can open up -- that's how I got good enough to get hired by Google!